ncm-authconfig/src/main/perl/authconfig.pm

```diff
@@ -2,28 +2,25 @@
 # ${developer-info}
 # ${author-info}
 
-#
-# authconfig - authconfig NCM component
-#
-################################################################################
-
 package NCM::Component::authconfig;
 
 use strict;
 use NCM::Component;
-use CAF::Process;
 use vars qw(@ISA $EC);
 @ISA = qw(NCM::Component);
 $EC=LC::Exception::Context->new->will_store_all;
+
```

Comment:
This looks to support NoAction mode (as in, it only use CAF libraries to do stuff) so can you mark it as NoActionSupported please?